src/amo/addonManager.js

```diff
@@ -192,7 +196,7 @@ export function addChangeListeners(
   |}) => void,
   mozAddonManager: MozAddonManagerType,
   { _log = log }: {| _log: typeof log |} = {},
-) {
+): (e: AddonChangeEvent) => void | Promise<void> {
```

Comment:
I think we could introduce a type like `HandleChangeEventFunction`

Reply:
and a question: why `void`? Shouldn't it be `Promise<void>` only?

Reply:
The type of the response is documented above as `boolean`, so I guess that should suffice.